nss-day-cohort-52/giffygram-wild-hurricanes
# Post form render button and close button functionality

#### Issues Completed
Applies to issue #25 

#### Changes Made
1.  [X] Feature
    [] Bug-fix
    [] Feature that will cause a bug
    [] Not a feature or a bug- requires documentation
2. Added file `filename` to `directoryname` directory.
Didn't create any new files
3. Modified file `filename` to include `functionalityname` and summary of functionality.
Modified GiffyGram.js and also PostEntry.js so that the post entry button is displayed and then when clicked on by the user  the post entry form should render. I also included the code to be able to close the entry form as well.
4. Make sure your code has been commented.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    commit your current code (git add --all, git commit -m "")
    git fetch --all
    git checkout csl-postFormRenderButton
    ```
2. Serve the code to browser.
3. Test app functionality.
    > How should the reviewer test functionality?
    > What is the intended outcome?
    > When the home page loads you should see the post entry button, when clicked it should render the post entry form. There is also a close button on the form which when clicked should close the entry form. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
